### DISCUSSION TITLE
Broken?

### DISCUSSION THREAD

**Discussion by CraigFreeze** [lukePeavey/quotable#212] [Report Downtime]
I can't connect to the rest API, I had it working previously. Is it down?

**Comment by arigl** [comment]
Yep, API is down. Should hopefully be back up soon.

**Reply by nikunj8866** [reply]
@arigl Is there any time estimate when it'll be back up?

**Comment by kartnagrale** [comment]
When it will come back?

**Comment by gsanchez2004** [comment]
There is any estimation on when it will come back ? thanks in advance !!!!
